sympy/polys/polytools.py

```diff
@@ -52,12 +54,32 @@
 
 
 
+def _sympifyit_Poly(func):
```

Comment:
How about `_polifyit`. (There already exists PolificationFailed.)

Reply:
Yes that seems better. I've changed it to `_polifyit`